common/constants.js

```diff
@@ -16,6 +16,7 @@ export const {
 export const PORTAL_ID = 'portal';
 export const DEFAULT_IDLE_TIMEOUT = 5 * 60 * 1000;
 export const MAX_SIZE_RANDOM_BUFFER = 60000;
+export const WORKER_DECRYPTION_BUFFER_SIZE = 10;
```

Comment:
Why we don't use navigator.hardwareConcurrency API?

Reply:
It's not a count of cpu cores and pool size. It's a buffer for decrypted items inside worker.